VIS/Areas/VIS/Scripts/model/callouts.js

```diff
@@ -17162,6 +17163,24 @@
                     }
                 }
                 mTab.setValue("M_Warehouse_ID", Util.getValueOfInt(dr["M_Warehouse_ID"]));
+
+                if (DataPrefix["VA077_"]) {
+                    mTab.setValue("VA077_HistoricContractDate", Util.getValueOfDate(dr["VA077_HistoricContractDate"]));
```

Comment:
No check if date is nulll, or is it confirmed that there will be date coming from model always